homework10/zanovo10/Table/TableViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  TableViewController.swift
+//  zanovo10
+//
+//  Created by qeqwe on 20.11.2023.
+//
+
+import UIKit
+
+final class TableViewController: UIViewController {
+    
+    private var settings = SourseTable.makeSettingsUnGroups()
+    
+    private let tableView: UITableView = .init(frame: .zero, style: .insetGrouped)
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupTableView()
+        tableView.register(TableViewCell.self, forCellReuseIdentifier: "TableViewCell")
+        tableView.dataSource = self
+        tableView.separatorStyle = .singleLine
+        tableView.separatorColor = .black
```

Comment:
```suggestion
        tableView.register(TableViewCell.self, forCellReuseIdentifier: "TableViewCell")
        tableView.dataSource = self
        tableView.separatorStyle = .singleLine
        tableView.separatorColor = .black
```
вынеси это в отдельный метод